site/zenodo_rdm/subcommunities/request.py

```diff
@@ -113,8 +117,111 @@ class ZenodoSubCommunityRequest(SubCommunityRequest):
 
     available_actions = {
         "delete": actions.DeleteAction,
-        "create": SubcommunityCreateAction,
         "cancel": actions.CancelAction,
+        # Custom implemented actions
```

Comment:
what do you mean by "custom implemented actions"? custom = only present in zenodo?

Reply:
Custom meaning that they aren't using the default `actions.Action`. Comment was added to match invenio-communities 
https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/subcommunities/services/request.py#L70